torchbenchmark/util/framework/huggingface/patch_hf.py

```diff
@@ -16,16 +16,17 @@ def cache_model(name: str):
     model_ctor.from_config(model_config)
 
 def patch_transformers():
-    import transformers
-    transformers_dir = os.path.dirname(transformers.__file__)
-    for patch_file in os.listdir(PATCH_DIR):
-        patch_file_fullpatch = os.path.join(PATCH_DIR, patch_file)
-        try:
-            subprocess.check_output(["patch", "-p1", "--forward", "-i", patch_file_fullpatch, "-r", "/tmp/rej"], cwd=transformers_dir)
-        except subprocess.SubprocessError as e:
-            output_str = str(e.output)
-            if "previously applied" in output_str:
-                return
-            else:
-                print(str(output_str))
-                sys.exit(1)
+    pass
+    # import transformers
+    # transformers_dir = os.path.dirname(transformers.__file__)
```

Comment:
I guess there is a better way than just commenting the code:
1. detect if PATCH_DIR exists, if not, directly return
2. check if `patch_file_fullpatch` is ending with `.patch`, if not, skip.

Reply:
solved by https://github.com/pytorch/benchmark/pull/1025/commits/33196c26e7eeb2c1529aca1c608d7c7bc971033f.